librz/main/rz-diff.c

```diff
@@ -196,55 +196,148 @@ typedef struct diff_hex_view_t {
 #define rz_diff_ctx_set_opt(x, o)  rz_diff_ctx_set_def(x, option, DIFF_OPT_UNKNOWN, o)
 
 static void rz_diff_show_help(bool usage_only) {
-	printf("Usage: rz-diff [options] <file0> <file1>\n");
+	printf("%s%s%s", Color_CYAN, "Usage: ", Color_RESET);
+	printf("rz-diff [options] <file0> <file1>\n");
 	if (usage_only) {
 		return;
 	}
+	const char *options[] = {
```

Comment:
You can add `// clang-format on` and `// clang-format off` to disable formatted between these "braces", it would allow manual formatting of that help that will roughly preserve the previous one.